csharpTasks/task1/MyVector/MyVector/Vector.cs

```diff
@@ -25,34 +25,54 @@ private void Foo()
 
         }        
         */
+        /// <summary>
+        /// Поля стуктуры гориз и вертикальная состовляющая
+        /// </summary>
+        public double X;
```

Comment:
обычно структуры делают иммутабельными, почему такое определение может этому помешать?

Reply:
потому что поле public
```suggestion
        private double _x;
```